myserver.h

```diff
@@ -0,0 +1,87 @@
+/*#ifndef MYSERVER_H
+#define MYSERVER_H
+
+#include <QTcpServer>
+#include <QTcpSocket>
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QJsonParseError>
+#include <QSqlDatabase>
+#include <QSqlQuery>
+#include <QFile>
+#include <QString>
+#include <QByteArray>
+
+class myserver: public QTcpServer{
+    Q_OBJECT
+public:
+    myserver();
+    ~myserver();
+
+    QTcpSocket* socket;
+    QByteArray Data;
+
+    QJsonDocument doc;
+    QJsonParseError docError;
+
+    QByteArray res;
+
+    QSqlDatabase db;
+
+
+public slots:
+    void startServer();
+    void incomingConnection(qintptr socketDescriptor);
+    void sockReady();
+    void sockDisc();
+};
+
+#endif // MYSERVER_H*/
+#ifndef MYSERVER_H
+#define MYSERVER_H
+
+#include <QTcpServer>
+#include <QTcpSocket>
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QJsonParseError>
+#include <QSqlDatabase>
+#include <QSqlQuery>
+#include <QFile>
+#include <QString>
+#include <QByteArray>
+#include <QByteArrayView>
+#include <QtCore>
+#include <iostream>
+#include <QIODevice>
+#include <string>
+#include <vector>
+#include <memory.h>
+#include <QRegularExpression>
+#include <QStringConverter>
+#include "lib/json.hpp"
```

Comment:
do you really need all this includes? Delete unnecessary. Leave those that you are using.

Reply:
Fixed it